api/v1/flow/flow.proto

```diff
@@ -481,7 +481,6 @@ enum DropReason {
     INVALID_VNI = 183;
     INVALID_TC_BUFFER = 184;
     NO_SID = 185;
-    MISSING_SRV6_STATE = 186 [deprecated = true];
```

Comment:
Nit: It probably really doesn't matter too much, as this seems to have been deprecated for a while now, but  I'd probably just have kept it here in the API as deprecated. We don't really gain much by removing this and it allows newer hubble CLI to still show this drop reason when connecting to older Cilium clusters. I don't think we ever removed any of the other deprecated drop reasons.

Also to be really pedantic, if you want to remove it, it would be cleaner to add a `reserved 186;` here, as the drop reasons shouldn't be reused.

But keeping it as is, is IMO also completely fine.